pyccl/baryons/vandaalen19.py

```diff
@@ -0,0 +1,94 @@
+__all__ = ("vanDaalen19",)
+
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class vanDaalen19(Baryons):
+    """The "van Daalen+ 2019" model boost factor for baryons.
+
+    .. note:: First presented in van Daalen et al., arXiv:1906.00968.
+
+              The boost factor is applied multiplicatively so that
+              :math:`P_{\\rm bar.}(k, a) = P_{\\rm DMO}(k, a)\\,
+              f_{\\rm vD19}(k, a)`.
+
+              Notice the model has only been tested at z=0 and is valid for
+              :math:`k\\leq 1 \\,h/{\\rm Mpc}.
+
+    Args:
+        fbar500c (:obj:`float`): the fraction of baryons in a halo within 
+        an overdensity of 500 times the critical density, given in units
+        of the ratio of :math:`\\Omega_b` to :math:`\\Omega_m`. 
+        Default to 0.7 which is approximately compatible with observations.
+        See Figure 16 of the paper.
+
+    """
+    name = 'vanDaalen19'
+    __repr_attrs__ = __eq_attrs__ = ("fbar500c")
+
+    def __init__(self, fbar500c=0.7):
```

Comment:
is 0.7 a reasonable default, or should we force people to choose a value?

Reply:
Yes, 0.7 is reasonable. See green band consistent with observations in their Fig 16.